zilencer/views.py

```diff
@@ -509,18 +509,57 @@ def __init__(self, reason: str) -> None:
         super().__init__(msg)
 
 
+class GcmPayload(TypedDict, total=False):
+    server: Optional[str]
+    realm_id: Optional[int]
+    realm_url: Optional[str]
+    realm_name: Optional[str]
+    user_id: Optional[int]
+    event: Optional[str]
+    zulip_message_id: Optional[int]
+    sender_id: Optional[int]
+    sender_email: Optional[str]
+    time: Optional[Union[int, float]]
+    mentioned_user_group_id: Optional[int]
+    mentioned_user_group_name: Optional[str]
+    recipient_type: Optional[str]
+    stream: Optional[str]
+    stream_id: Optional[int]
+    topic: Optional[str]
+    pm_users: Optional[str]
+    sender_avatar_url: Optional[str]
+    content: Optional[str]
+    content_truncated: Optional[bool]
+    sender_full_name: Optional[str]
+    zulip_message_ids: Optional[str]
+
+class RemoveGcmPayload(TypedDict, total=True):
+    event: str
+    zulip_message_ids: str
+
+class ApnsCustom(TypedDict):
+    zulip: Dict[str, Any]
```

Comment:
You should be able to specify a type for this, ideally using some shared common class that `GcmPayload` also inherits from